src/app/app.constants.ts

```diff
@@ -3,7 +3,8 @@ import {Theme} from './containers/app-header/app-header.component';
 
 const FeatureNames = {
   workAllocation: 'MC_Work_Allocation',
-  noticeOfChange: 'MC_Notice_of_Change'
+  noticeOfChange: 'MC_Notice_of_Change',
+  currentWAFeature: 'mc-work-allocation-active-feature'
```

Comment:
Why does this follow an entirely different naming convention to the other feature names?

Reply:
will ask Uday as this change is based off of his branch

Reply:
yes some of the feature names are small case in LD. There is not pattern. But, good point. It cannot be changed now as it affects all feature names for follow. In summary this comment is beyond the scope of this PR

Reply:
Lowercase vs Titlecase. Hyphens vs underscores.

Why can it not be changed now? It's just a flag in LaunchDarkly, isn't it?

Reply:
Actually, cancel this. It looks like the outliers are actually `MC_Work_Allocation` and `MC_Notice_of_Change`. The newly-added one is more consistent with the naming convention. In this particular context, it looks like the new one is the outlier but it's the other two that are wrong! 🤦‍♂️